src/Common/src/CoreWCF/Resources/Strings.resx

```diff
@@ -174,4 +174,7 @@
   <data name="CommunicationObjectCannotBeUsed" xml:space="preserve">
     <value>The communication object, {0}, is in the {1} state.  Communication objects cannot be used for communication unless they are in the Opened state.</value>
   </data>
+    <data name = "UnsupportedSecuritySetting" xml:space="preserve">
+    <value>The value '{1}' is not supported in this context for the binding security property '{0}'.</value>
```

Comment:
You added this to the wrong string resource file. It should have been added to the UDS specific one